public/characterCreator.html

```diff
@@ -1 +1,96 @@
-an interactive character creator 
\ No newline at end of file
+an interactive character creator
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]>      <html class="no-js"> <!--<![endif]-->
+<html>
+
+<head>
```

Comment:
all of the head should be covered by the nav

Reply:
You'll have to show us how this is supposed to go

Reply:
1. isn't Zack doing that bit, and 2. most likely through a mix of javascript and html imports